portfolio/src/main/webapp/style.css

```diff
@@ -28,7 +47,8 @@ body {
     line-height: 1.5;
     margin: 0 auto;
     max-width: 800px;
-    /* Padding: 2em 2em 4em; */
```

Comment:
According to the html/css style guide, you should prefer short hand whenever possible. I believe in this case it woud be Padding: 2em 0 4em

Reply:
Sorry, just 2em and 4em. No 0.